e2e-tests/insurance/cypress/e2e/journeys/declarations/modern-slavery/modern-slavery.spec.js

```diff
@@ -25,181 +26,172 @@ const {
-  let referenceNumber;
-  let url;
+context(
+  'Insurance - Declarations - Modern slavery page - As a UKEF legal adviser, I want to know whether an exporter adheres to the Modern Slavery Act 2015, So that UKEF is not at risk of supporting ethically unsustainable businesses',
+  () => {
+    let referenceNumber;
+    let url;
 
-  before(() => {
-    cy.completeSignInAndGoToApplication({}).then(({ referenceNumber: refNumber }) => {
-      referenceNumber = refNumber;
+    before(() => {
+      cy.completeSignInAndGoToApplication({}).then(({ referenceNumber: refNumber }) => {
+        referenceNumber = refNumber;
 
-      url = `${baseUrl}${INSURANCE_ROOT}/${referenceNumber}${MODERN_SLAVERY}`;
+        url = `${baseUrl}${INSURANCE_ROOT}/${referenceNumber}${MODERN_SLAVERY}`;
 
-      cy.navigateToUrl(url);
-      cy.assertUrl(url);
-    });
-  });
-
-  beforeEach(() => {
-    cy.saveSession();
-  });
-
-  after(() => {
-    cy.deleteApplication(referenceNumber);
-  });
-
-  it('renders core page elements', () => {
-    cy.corePageChecks({
-      pageTitle: CONTENT_STRINGS.PAGE_TITLE,
-      currentHref: `${INSURANCE_ROOT}/${referenceNumber}${MODERN_SLAVERY}`,
-      backLink: `${INSURANCE_ROOT}/${referenceNumber}${MODERN_SLAVERY}#`,
+        cy.navigateToUrl(url);
+        cy.assertUrl(url);
+      });
     });
-  });
 
-  describe('page tests', () => {
     beforeEach(() => {
-      cy.navigateToUrl(url);
+      cy.saveSession();
     });
 
-    it('renders a heading caption', () => {
-      cy.checkText(headingCaption(), CONTENT_STRINGS.HEADING_CAPTION);
+    after(() => {
+      cy.deleteApplication(referenceNumber);
     });
 
-    describe('intro', () => {
-      it('should render `answer the questions` copy', () => {
-        cy.checkText(answerTheQuestions(), CONTENT_STRINGS.INTRO.ANSWER_THE_QUESTIONS);
-      });
-
-      it('should render a `guiding principles` link', () => {
-        cy.checkLink(guidingPrinciplesLink(), '#', CONTENT_STRINGS.INTRO.GUIDING_PRINCIPLES_LINK.TEXT);
-      });
-
-      it('should render `if you say no` copy', () => {
-        cy.checkText(ifYouSayNo(), CONTENT_STRINGS.INTRO.IF_YOU_SAY_NO);
+    it('renders core page elements', () => {
+      cy.corePageChecks({
+        pageTitle: CONTENT_STRINGS.PAGE_TITLE,
+        currentHref: `${INSURANCE_ROOT}/${referenceNumber}${MODERN_SLAVERY}`,
+        backLink: `${INSURANCE_ROOT}/${referenceNumber}${MODERN_SLAVERY}#`,
       });
     });
 
-    describe('expandable details - definition of terms', () => {
+    describe('page tests', () => {
       beforeEach(() => {
         cy.navigateToUrl(url);
       });
 
-      it('should render summary text with collapsed conditional `details` content', () => {
-        const expectedText = CONTENT_STRINGS.EXPANDABLE.VERSIONS[0].INTRO;
-
-        cy.checkText(expandable.summary(), expectedText);
-
-        expandable.details().should('not.have.attr', 'open');
+      it('renders a heading caption', () => {
```

Comment:
should render

Reply:
We should keep it aligned with everything else. Created EMS-4069.